SWTableViewCell/PodFiles/SWTableViewCell.m

```diff
@@ -476,6 +476,23 @@ - (void)hideUtilityButtonsAnimated:(BOOL)animated
     }
 }
 
+- (void)hideUtilityButtons:(UITableView *)tableView exceptForCell:(SWTableViewCell *)cell animated:(BOOL)animated
+{
+	NSArray *paths = [tableView indexPathsForVisibleRows];
+    
+    NSMutableSet *visibleCells = [[NSMutableSet alloc] init];
+    
+    for (NSIndexPath *path in paths) {
+        [visibleCells addObject:[tableView cellForRowAtIndexPath:path]];
```

Comment:
An improvement would be to change this line to this.
```
SWTableViewCell *cellForRow = (SWTableViewCell *)[tableView cellForRowAtIndexPath:path];
if (cellForRow)
{
    [visibleCells addObject:cellForRow];
}
```
Without this change I've gotten the code to throw an exception here when `hideUtilityButtons:exceptForCell:animated:` is called in `swipeableTableViewCellDidEndScrolling:` and quickly showing the utility buttons and scrolling fast off the screen.